OtripleS.Web.Api/Services/TeacherAttachments/TeacherAttachmentService.cs

```diff
@@ -59,5 +59,18 @@ public partial class TeacherAttachmentService : ITeacherAttachmentService
 
             return storageTeacherAttachment;
         });
+
+        public ValueTask<TeacherAttachment> RemoveTeacherAttachmentByIdAsync(Guid teacherId, Guid attachmentId) =>
+        TryCatch(async () =>
+        {
+            ValidateTeacherAttachmentIdIsNull(teacherId, attachmentId);
```

Comment:
`Guid`s can never be null.
You probably want to call this method `ValidateTeacherAttachmentIds(...)`